docs/google/devices.md

```diff
@@ -0,0 +1,194 @@
+# Google Devices
+
+This section describes the devices in Cirq for Google hardware devices and their usage.
+Since quantum hardware is an active area of research, hardware specifications and best
+practices are constantly evolving in an attempt to continuously improve performance.
+While this information should be a solid base for beginning your quantum application,
+please work with your Google sponsor to obtain the latest information on devices that
+you plan to use. 
+
+## General limitations
+
+Qubits on Google devices are laid out in a grid structure.  Connectivity is limited to
+adjacent qubits, either horizontally or vertically.  
+
+Measurement takes much longer than other gates.  Currently, the only supported
+configuration is to have terminal measurement in the final moment of a circuit.
+
+Most devices have a limited number of gates that can applied to the device.  Gates not
```

Comment:
"number of gates" here might be confused with how many gates can be applied in succession in a given circuit. Might be better to say "set of gates". Could separately note that there is also an overall restriction on the length of the circuit.

Reply:
missing "be"

Reply:
applied on the device? applied with the device? "to" sounds ood

Reply:
Done.